test/src/e2e_vm_tests/test_programs/should_pass/language/references/dereferencing_operator_dot_on_structs/test.toml

```diff
@@ -1,4 +1,4 @@
 category = "run"
 expected_result = { action = "return", value = 42 }
-validate_abi = true
+validate_abi = false
```

Comment:
Why did this change? I see it's changed for many of the test programs.

Reply:
The tests are actually not checking/stressing/testing ABI at all. Like expected functions, types, configurables, etc. There is nothing special in the ABIs, just `main` function and used types.

The question is actually why the `validate_abi` was set to `true` at all :wink: Well, as usually the fact, I got the first test back then by copy-pasting some existing one that was by chance validating ABI and I didn't remove the validation...